homeassistant/components/wemo.py

```diff
@@ -91,6 +92,7 @@ def discovery_dispatch(service, discovery_info):
         if device is None:
             device = pywemo.discovery.device_from_description(url, None)
 
-        discovery_info = (device.name, device.model_name, url, device.mac)
+        discovery_info = (device.name, device.model_name, url, device.mac,
```

Comment:
Any time we can remove something from the system because it is no longer needed is a win.